views/matrixsplit.html

```diff
@@ -592,10 +590,10 @@
           \end{bmatrix}
           =
           \begin{bmatrix}
-            z^3 \cdot P_4 - 3 \cdot z^2 \cdot (z-1) \cdot P_3 + 3 \cdot z \cdot (z-1)^2 \cdot P_2 - (z-1)^3 \cdot P_1 \\
-            z^2 \cdot P_4 - 2 \cdot z \cdot (z-1) \cdot P_3 + (z-1)^2 \cdot P_2 \\
-            z \cdot P_4 - (z-1) \cdot P_3 \\
-            P_4
+            (1-z)^3 \cdot P_1 + 3 \cdot (1-z)^2 \cdot z \cdot P_2 + 3 \cdot (1-z) \cdot z^2 \cdot P_3 + z^3 \cdot P_4 \\
```

Comment:
no real reason to introduce all this whitespace, HTML strips it, as does MathJax, as does LaTeX when it processes it.